maat-orchestration/src/main/java/uk/gov/justice/laa/crime/orchestration/mapper/ProceedingsMapper.java

```diff
@@ -28,8 +29,16 @@ public class ProceedingsMapper extends CrownCourtMapper {
     public ApiUpdateApplicationRequest workflowRequestToUpdateApplicationRequest(
             ApplicationDTO application, UserDTO userDTO) {
 
-        ApiUpdateApplicationRequest updateApplicationRequest = new ApiUpdateApplicationRequest()
-                .withRepId(NumberUtils.toInteger(application.getRepId()))
+        ApiUpdateApplicationRequest request = new ApiUpdateApplicationRequest();
+        mapToApiProcessRepOrderRequest(application, request, Boolean.TRUE);
+        mapToApiUpdateApplicationRequest(application, userDTO, request);
+
+        return request;
+    }
+
+    private void mapToApiProcessRepOrderRequest(ApplicationDTO application, ApiProcessRepOrderRequest request, boolean isMeansAssessment) {
```

Comment:
I don't understand why we need the `isMeansAssessment` parameter? There doesn't need to be any branching logic around the status field, we should just use the new method `getCurrentStatus`